src/Collector/Stack.php

```diff
@@ -277,6 +287,38 @@ public function setResponseCode($responseCode)
         $this->responseCode = $responseCode;
     }
 
+    /**
+     * @return string
+     */
+    public function getDebugToken()
+    {
+        return $this->debugToken;
+    }
+
+    /**
+     * @param string $debugToken
+     */
+    public function setDebugToken($debugToken)
```

Comment:
please write this as `setDebugToken(string $debugToken): void` (and then we can remove the `@param` because it does not add any information anymore.